test/Microsoft.AspNetCore.Mvc.Core.Test/Builder/MvcAreaRouteBuilderExtensionsTest.cs

```diff
@@ -209,7 +209,7 @@ public void MapAreaRoute_ReplacesValuesForArea()
                 kvp =>
                 {
                     Assert.Equal(kvp.Key, "area");
-                    Assert.IsType<RegexRouteConstraint>(kvp.Value);
+                    Assert.IsType<IntRouteConstraint>(kvp.Value);
```

Comment:
The clue is to look at the name of the test `MapAreaRoute_ReplacesValuesForArea`

@iscifoni please rename the test to something more fitting because the constraints don't get replaced anymore :+1: